ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +72,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_replica_promotion.py::TestReplicaConn
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *ad_master_1replica_1client
```

Comment:
You will need to add this test in the nightly definitions in nightly_latest_selinux.yaml nightly_latest_testing_selinux.yaml nightly_latest_testing.yaml nightly_latest.yaml nightly_previous.yaml and nightly_rawhide.yaml